docs/data/sources/shopify.md

```diff
@@ -5,66 +5,39 @@ description: Amplitude’s app with Shopify is a smart analytics app that automa
 
 Amplitude’s app with Shopify is a smart analytics app that automates eCommerce tracking. With Amplitude’s app, you can automatically track and ingest key e-commerce events, event properties, and user properties on a Shopify or Shopify Plus store.
 
-## Considerations
+### Client-side (device mode) tracking
+During the installation process, Amplitude Shopify App adds the code snippet to all pages (included in theme.liquid) on your Shopify store. The benefits of this approach include:
+- Amplitude’s JS SDK library is loaded on all pages, except for the checkout
+
+### Server-side (cloud mode) tracking
+During the Amplitude connection setup, our Shopify App also adds a set of webhooks to your Shopify store. When a customer interacts with your store these changes are relayed server-side from Shopify to Amplitude. The advantages to this approach are:
+- 100% event capture for adds to cart, checkout steps, sales.
+- Customer data (for example, email) securely relayed server-side.
+- No extra scripts on the sensitive and secure checkout pages.
+- Accurate marketing attribution, even when customers use ad-blockers or cookie opt-outs.
+- More reliable and trustyworthy than client-side events (as it represents final, concluded transactions or states within Shopify).
```

Comment:
```suggestion
- More reliable and trustworthy than client-side events because it represents final, concluded transactions and states within Shopify.
```